### DISCUSSION TITLE
Compile Error

### DISCUSSION THREAD

**Discussion by tach-tm** [TrinityCore/TrinityCore#27120] [Q&A] [answered]
Error C2664 "boost::asio::basic_waitable_timerstd::chrono::steady_clock,boost::asio::wait_traits,boost::asio::any_io_executor::basic_waitable_timer(const boost::asio::any_io_executor &,const std::chrono::time_pointstd::chrono::steady_clock,std::chrono::nanoseconds &)": cannot convert argument 1 from "boost::asio::executor" in "const boost::asio::any_io_executor &" worldserver worldserver\TCRest\server_http.h 296

here is such an error to come out when compilin
Please help!
boots = 1.78
VS2019
![Uploading sshot-2.jpg…]()
g

**Comment by Shauren** [comment, marked as answer]
We do not have a `TCRest\server_http.h` file in any branch, therefore this is not a TrinityCore issue
